service/src/config.rs

```diff
@@ -19,32 +20,46 @@ impl Config {
 		node_port: String,
 		worker_ip: String,
 		worker_rpc_port: String,
+		untrusted_worker_port: String,
 		worker_mu_ra_port: String,
 	) -> Self {
 		Self {
 			node_ip,
 			node_port,
 			worker_ip,
 			worker_rpc_port,
+			untrusted_worker_port,
 			worker_mu_ra_port,
-			ext_api_url: None,
 		}
 	}
 
+	/// Gives back the client url for the node (including ws://)
 	pub fn node_url(&self) -> String {
 		format!("{}:{}", self.node_ip, self.node_port)
 	}
 
-	pub fn worker_url(&self) -> String {
+	pub fn trusted_worker_url(&self) -> String {
 		format!("{}:{}", self.worker_ip, self.worker_rpc_port)
 	}
 
+	pub fn trusted_worker_url_for_client(&self) -> String {
```

Comment:
We had quite the mess with some `wss` and `ws` values in the main file (including comments). With these functions, I intend to define the ws addresses the client uses in one place only.

Reply:
Would it make sense to call also the field self.trusted_worker_rpc_port to be consistent with the method name and the methods blow?

Reply:
This should only be a comments issue. The worker can never know if it'is behind wss or ws because that depends on if it's running behind an ngnx instance. So hardcoded wss is wrong as far as I know.

Reply:
The worker registers itself on the pallet teerex with an url. Before it was "ws", but that doesn't work, because we're using wss internal. Also without nginx. Until now it was not a problem because nothing depended on the teerex url, but that's different now. And I run into that problem. What would you suggest instead?

Reply:
Ah I see, I was not aware that we always have `wss` now, but it makes sense after the TLS change in the enclave. Then your change makes sense.